discovery-provider/src/api/v1/tracks.py

```diff
@@ -253,8 +274,8 @@ def get(self):
 class FullTrackByRoute(Resource):
```

Comment:
should FullTrackByRoute be renamed bulk?

Reply:
probably! as an aside: why do we use Resources at all? Couldn't these all just be methods? It's not like we have post/put/delete requests for a route or are mapping to actual resources